js/src/cli/apps.ts

```diff
@@ -19,7 +19,10 @@ export default class AppsCommand {
   constructor(program: Command) {
     this.program = program;
 
-    const command = this.program.command("apps");
+    const command = this.program.command("apps").option(
+      "--enabled",
+      "Only show enabled apps"
```

Comment:
Consider adding a more descriptive help message for the --enabled flag, such as \"Filter to show only enabled apps in the list\"